examples/protocol_specification_ex/http.yaml

```diff
@@ -19,3 +19,12 @@ speech_acts:
     headers: pt:str
     bodyy: pt:bytes
 ...
+---
+entry: [request]
+reply:
+  request: [response]
+  response: []
+exit: [response]
+roles: {client, server}
+end_states: [successful]
```

Comment:
Looks fine. Always a two message dialogue: `request`, `response`

Reply:
Yes I think the former makes more sense in light of the RESTfullness (https://restfulapi.net/statelessness/)